src/internal/utils.ts

```diff
@@ -133,7 +133,7 @@ export function initProperty(name: string, key: string, whatis: WhatIsIt) {
       schemas.get(name)[key] = {};
       break;
     default:
-      throw new TypeError('"whatis" is not supplied OR dosnt have a case yet!');
+      throw new TypeError('"whatis" is not supplied OR doesn't have a case yet!');
```

Comment:
sorry, but adding this `'` here breaks the string

```suggestion
      throw new TypeError('"whatis" is not supplied OR doesn\'t have a case yet!');
```

Reply:
Good catch! I made a lot of manual fixes, but also some search&replaces, and didn't notice that.